pyomo/contrib/pynumero/extensions/utils.py

```diff
@@ -15,6 +15,11 @@
 def find_pynumero_library(library_name):
 
     asl_path = find_library(library_name)
+    if asl_path is not None:
+        return asl_path
+
+    # On windows the library is prefixed with 'lib'
+    asl_path = find_library('lib'+library_name)
```

Comment:
This is minor and should not hold-up the merge, however, it is my understanding that the pynumero library can or  does provide more than ASL interfaces.

Reply:
Yeah, the local variable name in that function is unnecessarily specific. I can make that quick change. Currently the `pynumero_libraries` package only includes ASL.